posthog/api/test/test_feature_flag.py

```diff
@@ -6077,7 +6078,7 @@ def test_user_blast_radius_with_single_cohort(self):
                             "type": "OR",
                             "values": [
                                 {"key": "group", "value": "none", "type": "person"},
-                                {"key": "group", "value": [1, 2, 3], "type": "person"},
+                                {"key": "group", "value": ["1", "2", "3"], "type": "person"},
```

Comment:
What's interesting is I expected this to fail because the filter values are strings and the group values are integers, but it passes.

In fact, if I make the filter values integers, then this fails. @aspicer do you have any insights into why that is?